### ISSUE TITLE
card - accessibility issues

### ISSUE THREAD

**Issue by JoaoEnesGoncalves** [lumada-design/hv-uikit-react#726]
**Describe the bug**
1. sub header have insufficient contrast
2. checkbox without label
3. button "..." does not have a name available to an accessibility API
4. div role="button" e label "..."  does not have a name available to an accessibility API

**To Reproduce**
1. Run 
`pa11y --runner htmlcs --runner axe --reporter json --standard WCAG2AA --root-element "#root > div.Component-content-4" http://localhost:9002/iframe.html?id=corecard--card2`

**Additional context**

**❌ 1- sub header have insufficient contrast**
```
http://localhost:9002/iframe.html?id=corecard--card2
  "code": "WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail",
  "type": "error",
  "typeCode": 1,
  "message": "This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.85:1. Recommendation:  change text colour to #767676.",
  "context": "<span style="border - right: 1 px solid rgb(222, 222, 222);padding - right: 10 px;margin - right: 10 px; ">Just now</span>",
  "selector": "#root > div:nth-child(2) > div > div > div > div > div:nth-child(1) > div:nth-child(1) > span:nth-child(2) > div > span:nth-child(1)",
  "runner": "htmlcs",

  "code": "WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail",
  "type": "error",
  "typeCode": 1,
  "message": "This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.85:1. Recommendation:  change text colour to #767676.",
  "context": "<span>L20</span>",
  "selector": "#root > div:nth-child(2) > div > div > div > div > div:nth-child(1) > div:nth-child(1) > span:nth-child(2) > div > span:nth-child(2)",
  "runner": "htmlcs",

  "code": "color-contrast",
  "type": "error",
  "typeCode": 1,
  "message": "Elements must have sufficient color contrast (https://dequeuniversity.com/rules/axe/3.3/color-contrast?application=axeAPI)",
  "context": "<span style="
  border - right: 1 px solid rgb(222, 222, 222);padding - right: 10 px;margin - right: 10 px;
  ">Just now</span>",
  "selector": "#root > div:nth-child(2) > div > div > div > div > div:nth-child(1) > div:nth-child(1) > span:nth-child(2) > div > span:nth-child(1)",
  "runner": "axe",
  "runnerExtras": {
    "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
    "impact": "serious",
    "help": "Elements must have sufficient color contrast",
    "helpUrl": "https://dequeuniversity.com/rules/axe/3.3/color-contrast?application=axeAPI"
  }
```

**❌ 2- checkbox without label**
```
  http://localhost:9002/iframe.html?id=corecard--card2
  "code": "label",
  "type": "error",
  "typeCode": 1,
  "message": "Form elements must have labels (https://dequeuniversity.com/rules/axe/3.3/label?application=axeAPI)",
  "context": <input class="MuiPrivateSwitchBase - input - 408" type="checkbox" data-indeterminate="false" value=""  value >,
  "selector": "#root > div:nth-child(2) > div > div > div > div > div:nth-child(4) > div:nth-child(1) > label > span:nth-child(1) > span > input",
  "runner": "axe",
  "runnerExtras": {
    "description": "Ensures every form element has a label",
    "impact": "critical",
    "help": "Form elements must have labels",
    "helpUrl": "https://dequeuniversity.com/rules/axe/3.3/label?application=axeAPI"
  }
```

**❌ 3- button "..." does not have a name available to an accessibility API**
```
  http://localhost:9002/iframe.html?id=corecard--card2
  "code": "WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name",
  "type": "error",
  "typeCode": 1,
  "message": "This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined.",
  "context": "<button class="
  MuiButtonBase - root - 44 MuiButton - root - 18 HvButton - root - 9 MuiButton - text - 20 MuiButton - textPrimary - 21 HvButton - ghostSecondary - 16 MuiButton - flat - 23 MuiButton - flatPrimary - 24 MultipleActionsWithMediaButtons - smallButton - 416 " tabindex="
  0 " type...",
  "selector": "#root > div:nth-child(2) > div > div > div > div > div:nth-child(4) > div:nth-child(2) > button:nth-child(2)",
  "runner": "htmlcs",

  "code": "button-name",
  "type": "error",
  "typeCode": 1,
  "message": "Buttons must have discernible text (https://dequeuniversity.com/rules/axe/3.3/button-name?application=axeAPI)",
  "context": "<button class="
  MuiButtonBase - root - 44 MuiButton - root - 18 HvButton - root - 9 MuiButton - text - 20 MuiButton - textPrimary - 21 HvButton - ghostSecondary - 16 MuiButton - flat - 23 MuiButton - flatPrimary - 24 MultipleActionsWithMediaButtons - smallButton - 416 " tabindex="
  0 " type...",
  "selector": "#root > div:nth-child(2) > div > div > div > div > div:nth-child(4) > div:nth-child(2) > button:nth-child(2)",
  "runner": "axe",
  "runnerExtras": {
    "description": "Ensures buttons have discernible text",
    "impact": "critical",
    "help": "Buttons must have discernible text",
    "helpUrl": "https://dequeuniversity.com/rules/axe/3.3/button-name?application=axeAPI"
  }
```

**❌ 4- div role="button" e label "..."  does not have a name available to an accessibility API**
```
http://localhost:9002/iframe.html?id=corecard--card9
"code": "WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Div.Name",
"type": "error",
"typeCode": 1,
"message": "This element has role of button but does not have a name available to an accessibility API. Valid names are: element content, aria-label undefined, aria-labelledby undefined.",
"context": "<div role=\"button\" tabindex=\"0\" class=\"DropDownMenu - icon - 380 WithStyles - DropDownMenu--icon - 378 \"><svg xmlns=\"  http: //www.w3.org/2...</div>",
"selector":"#root > div:nth-child(2) > div > div > div > div > div:nth-child(2) > div:nth-child(2) > div > div:nth-child(2) > div > div",
"runner":"htmlcs",
"runnerExtras":{}},
```
